lib/src/log.rs

```diff
@@ -129,22 +130,84 @@ pub fn reset_task_log_directory(path: &Path) -> Result<(), Error> {
     Ok(())
 }
 
+pub fn read_last_lines_as_byte_deque(file: &mut File, lines: usize) -> Result<VecDeque<u8>, Error> {
+    // The integer types were chosen to avoid overflows when converting between signed and unsigned
+    // integers. u16 is used here so that we can support 16bit platforms with simple integer casts
+    // (FIXME Not sure if pueue even is supported on any 16bit platforms)
+    let chunk_size = 4096u16;
+    let mut buf = vec![0u8; chunk_size as usize];
+    let size = match file.seek(io::SeekFrom::End(0)) {
+        Ok(p) => p,
```

Comment:
Please don't use single-character variable names.
Especially if they're rather confusing. Please rename `p` to `size` or `seeked_bytes`.